connect/reports/renderers/base.py

```diff
@@ -1,6 +1,15 @@
 #  Copyright © 2021 CloudBlue. All rights reserved.
 
 from abc import ABCMeta, abstractmethod
+import zipfile
+import os
+from datetime import datetime
+import tempfile
+import json
+
+import pytz
+
+NOW = datetime.now()
```

Comment:
You cannot set now as a constant here. It will be evaluated when the module is imported.